src/main/java/com/erigitic/sql/TESqlManager.java

```diff
@@ -130,31 +153,38 @@ public Connection getJobsConnection() {
         return jobsConnection;
     }
 
+    /**
+     * Returns the current connection for {@link org.spongepowered.api.command.spec.CommandExecutor}s.
+     * DO NOT CACHE!
+     */
+    public Connection getCommandConnection() {
+        return commandConnection;
+    }
+
     /**
      * Closes all connections and resets the fields to `null`.
      */
     @Override
     public void close() {
         dataSource = null;
 
-        try {
-            if (accountsConnection != null && !accountsConnection.isClosed()) {
-                accountsConnection.close();
-            }
-        } catch (SQLException e) {
-            logger.warn("Failed to close accounts connection!", e);
-        }
-
+        suppressedCloseConnectionIfNotNull(accountsConnection, "Failed to close accounts connection!");
         accountsConnection = null;
 
+        suppressedCloseConnectionIfNotNull(jobsConnection, "Failed to close jobs connection!");
+        jobsConnection = null;
+
+        suppressedCloseConnectionIfNotNull(commandConnection, "Failed to close commands connection!");
+        commandConnection = null;
+    }
+
+    private void suppressedCloseConnectionIfNotNull(Connection accountsConnection, String s) {
```

Comment:
Should the `accountsConnection` variable be renamed to something more general such as `connection`?

Reply:
Not really as the ``accountsConnection`` is named that way in order to tell it should be used for querying account information such as balances or settings. Renaming it to just ``connection`` would falsely suggest a general purpose connection in my opinion.

Reply:
It seems weird though as when this function is called on lines 171, 174 and 177 it takes in arguments such as `accountsConnection`, `jobsConnection`, and `commandConnection` respectively. So it isn't only taking in an account connection.

Reply:
Oh I see. I was thinking you meant the entire variable+field and not that you're talking about that one method parameter ^^``  
In that case you're right oc. I'll get that changed.